libs/features/interlinks.js

```diff
@@ -55,7 +57,7 @@ export default async function interlink(path, limit = 1000) {
       const paraLinks = p.querySelectorAll('a').length;
       const paraWords = p.textContent.split(/\s/).length;
       const maxParaLinks = Math.floor(paraWords / 40) - paraLinks;
-      if (maxParaLinks > 0) {
+      if (exceptionLanguages.includes(language) || maxParaLinks > 0) {
```

Comment:
minor thing, but since you're checking multiple times whether the language is part of exceptionLanguages (lines 39 and 60), maybe you can rewrite the code as:
```js
// line 27
const isExceptionLanguage = ['zh', 'ko', 'ja', 'th', 'he'].includes(language);

// line 39
const wordBorder = isExceptionLanguage ? '' : '\\b';

// line 60
if (isExceptionLanguage || maxParaLinks > 0) {
```

Reply:
Done.